### ISSUE TITLE
Is this plugin now compatible with LS 4.1.1?

### ISSUE THREAD

**Issue by NZnick** [olleharstedt/MassAction#17]
Currently the readme for this plugin states “changes to this plugin are NOT compatible with any released version of LimeSurvey” and that the plugin requires version 4.0.0dev to run.

The latest stable version of LS is now 4.1.1. Are the changes in this repository compatible with this version (and subsequent versions) of LS? If so, should the readme be updated to show this?

**Comment by olleharstedt** [comment]
No idea. Test it?

**Comment by olleharstedt** [comment]
@NZnick Did you test?

**Comment by NZnick** [comment]
No, not yet. I meant to, but life got hectic, and then I forgot. I'll do it by the end of the week and let you know how it goes.

**Comment by olleharstedt** [comment]
It's OK, maybe I can find time to test it too.

**Comment by olleharstedt** [comment]
Works kind-of in LimeSurvey 5, but not for localized fields like question and help.
